packages/components/src/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -261,6 +257,12 @@ export const ConfirmationModal = forwardRef(function ConfirmationModalInternal(
 
   function handleAction(type: "confirm" | "cancel") {
     return () => {
+      if (type === "confirm") {
+        state.onConfirm && state.onConfirm();
+      } else if (type === "cancel") {
+        state.onCancel && state.onCancel();
+      }
```

Comment:
```
      if (type === "confirm") {
        state.onConfirm?.();
      } else if (type === "cancel") {
        state.onCancel?.();
      }
```
We can simplify this with optional chaining.

Reply:
Simplified in b5a4b40eb7cf754abf35c536ae2ddc6856e0b05a